api/app/routers/hfi_calc.py

```diff
@@ -0,0 +1,72 @@
+""" Routers for HFI Calculator """
+import logging
+from typing import List
+from fastapi import APIRouter, Response, Depends
+import app
+from app import wildfire_one
+from app.auth import authentication_required, audit
+from app.schemas.hfi_calc import HFIWeatherStationsResponse, WeatherStationProperties,\
+    FuelType, FireCentre, PlanningArea, WeatherStation
+from app.db.crud.hfi_calc import get_fire_centre_by_id, get_fuel_type_by_id,\
+    get_planning_area_by_id, get_planning_weather_stations
+
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(
+    prefix="/hfi-calc",
+    dependencies=[Depends(authentication_required), Depends(audit)]
+)
+
+
+@router.get('/', response_model=HFIWeatherStationsResponse)
```

Comment:
I think we should go ahead and have this hang under '/hfi-calc/fire_centres' or somthing from the get go? Doesn't feel right to me having the fire centre list hanging directly from /hfi-calc